L1TriggerConfig/L1ScalesProducers/src/L1ScalesTester.cc

```diff
@@ -3,82 +3,74 @@
 #include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/Framework/interface/ESHandle.h"
 
-#include "CondFormats/L1TObjects/interface/L1CaloEtScale.h"
-#include "CondFormats/DataRecord/interface/L1EmEtScaleRcd.h"
-#include "CondFormats/DataRecord/interface/L1JetEtScaleRcd.h"
-#include "CondFormats/L1TObjects/interface/L1CaloEcalScale.h"
-#include "CondFormats/L1TObjects/interface/L1CaloHcalScale.h"
-#include "CondFormats/DataRecord/interface/L1CaloEcalScaleRcd.h"
-#include "CondFormats/DataRecord/interface/L1CaloHcalScaleRcd.h"
-
 #include <iostream>
 
 using std::cout;
 using std::endl;
 
-L1ScalesTester::L1ScalesTester(const edm::ParameterSet& ps) { cout << "Constructing a L1ScalesTester" << endl; }
-
-L1ScalesTester::~L1ScalesTester() {}
+L1ScalesTester::L1ScalesTester(const edm::ParameterSet& ps)
+    : emScaleToken_(esConsumes()),
+      ecalScaleToken_(esConsumes()),
+      hcalScaleToken_(esConsumes()),
+      jetScaleToken_(esConsumes()) {
+  cout << "Constructing a L1ScalesTester" << endl;
```

Comment:
Is this `cout` thread safe? my understanding was the `cout`-s should be avoided, and the relevant MsgLogger is to be used.

Reply:
`cout` will not lead to crashes but output sent to standard out is likely to be intermixed with other output.

The purpose of this pull request is to remove the deprecation warnings (since Core wants to eventually remove the deprecated interface). As such, I'd prefer the actual maintainers of the packages to do any work beyond what is needed to accomplish the Core goals.

Reply:
I agree @Dr15Jones 

@cms-sw/l1-l2 will you take care of this change in a follow-up PR?